sig-contributor-experience/meeting-notes-archive/2019-meeting-notes.md

```diff
@@ -2824,7 +2824,7 @@ Agenda:
         *   [Sig-testing’s deploy policy](https://github.com/kubernetes/community/blob/master/sig-testing/charter.md#deploying-changes)
         *   [Sig-contribex’s deploy policy](https://github.com/kubernetes/community/blob/master/sig-contributor-experience/charter.md#cross-cutting-and-externally-facing-processes)
         *   Dec 10 not sure if this is the root cause PR? [https://github.com/kubernetes/test-infra/pull/10435](https://github.com/kubernetes/test-infra/pull/10435) 
-        *   Dec 10 notify k-dev@ of k/community change [https://groups.google.com/forum/#!msg/kubernetes-dev/oWbV6uiQMIM/0ku8cvSZBgAJ](https://groups.google.com/forum/#!msg/kubernetes-dev/oWbV6uiQMIM/0ku8cvSZBgAJ) 
+        *   Dec 10 notify dev@kubernetes.io of k/community change [https://groups.google.com/forum/#!msg/kubernetes-dev/oWbV6uiQMIM/0ku8cvSZBgAJ](https://groups.google.com/forum/#!msg/kubernetes-dev/oWbV6uiQMIM/0ku8cvSZBgAJ) 
```

Comment:
this file does not need to be changed as its an archive so we should keep the history